src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -242,6 +251,29 @@ services.AddHttpClient(
 Note: The single instance returned by `HttpClientFactory` is reused by all
 export requests.
 
+## Experimental features
+
+The OTLP exporter contains the following experimental features that are
+available only in pre-release packages:
+
+* The [exemplar
+  specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#exemplar)
+  is not yet stable. Pre-release builds of the OpenTelemetry .NET SDK and the
+  OTLP exporter support sampling and exporting exemplars.
+* Support for exporting the following fields on `LogRecord`: `CategoryName`,
+  `EventId`, and `Exception`. These fields are unique to OpenTelemetry .NET's
+  data model (i.e., they are not reflected in the [standard data
+  model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#log-and-event-record-definition)).
+  While they can be exported as attributes, there is no established conventions
+  for the attribute key names. In pre-release builds, these fields are exported
+  as attributes mapped as follows:
+  * `CategoryName` maps to `dotnet.ilogger.category_name`.
+  * `EventId` maps to two attributes `Id` and `Name` representing `Event.Id` and
+    `EventId.Name`, respectively.
```

Comment:
Why is this `EventId.Name` instead of `Event.Name` (it seems super weird comparing with `Event.Id`)?

Reply:
it is referring to https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.eventid?view=dotnet-plat-ext-7.0

Reply:
I know but the name we have here sounds wrong.

Reply:
There was a typo which I fixed. It should be `EventId.Id` not `Event.Id`

Reply:
> There was a typo which I fixed. It should be `EventId.Id` not `Event.Id`. Sorry for the confusion

It's getting even worse.

Reply:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.eventid.id?view=dotnet-plat-ext-7.0

Reply:
I think @reyang is asking about the "attribute" name?
The attribute names are just "Id" and "Name" as per this doc.